servicex/templates/requests_table.html

```diff
@@ -47,6 +48,16 @@
           </div>
         </td>
         <td id="replicas-{{ req.request_id }}">-</td>
+        <td>
+          {% if req.incomplete %}
+            <button id="cancel-{{ req.request_id }}"
+                    onclick="fetch('/servicex/transformation/{{ req.request_id }}/cancel').then((res) => location.reload())"
```

Comment:
With this implementation, we are hitting a Flask backend API endpoint from a Flask frontend webpage. 

I chose this over two other possibilities:
- Place the cancel logic in a web route instead of an API resource (not desirable because then other frontends such as the Python client cannot trigger cancellation)
- Create a web route which wraps the API resource (felt a bit complicated)

The reload is done to remove the cancel button and update the status column, but we could use more javascript to avoid it if needed.